<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-7IDTCRC" failures="1" tests="5" name="Login Test" time="4.875" errors="0" timestamp="2022-02-25T23:43:23 EST">
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="getTitleTest" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [Login | Salesforce] but found [Login | Salesforce1]">
      <![CDATA[java.lang.AssertionError: expected [Login | Salesforce] but found [Login | Salesforce1]
at com.SalesForceLoginTests.NegativeLoginTest.getTitleTest(NegativeLoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- getTitleTest -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="userNameButton" time="0.323"/>
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="passwordButton" time="0.296"/>
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="signinButton" time="0.301"/>
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="homePage" time="0.0">
    <skipped/>
  </testcase> <!-- homePage -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="loginTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Login Test -->
